tests/store/test_datasource.py

```diff
@@ -536,6 +561,7 @@ def test_key_date_compare():
     with pytest.raises(ValueError):
         in_date = d.key_date_compare(keys=error_key, start_date=start, end_date=end)
 
+
```

Comment:
@gareth-j Can't add a detail on the specific line L204 as this was not changed but the test is checking for a folder called "datasource/uuid/..." but this now seems to have changed to "data/uuid/..." - is this expected behaviour for the changes made?

Reply:
Same issue for `test_in_daterange()` failing test on L493 - expected "datasource" not "data" as part of the uuid (filepath)

Reply:
Yeah I wasn't sure about those changes, I'll take a look at the git blame and see what happened.

Reply:
So this may be related to the changes in the way the Datasource was written out within the save() method? This used to do something internal but now primarily uses `openghg.objectstore` instead.

e.g. [Datasource.save() on devel](https://github.com/openghg/openghg/blob/devel/openghg/store/base/_datasource.py#L546) versus [Datasource.save() on new_object_store](https://github.com/openghg/openghg/blob/new_object_store/openghg/store/base/_datasource.py#L625)